docs/workflow/testing/libraries/testing-wasm.md

```diff
@@ -0,0 +1,49 @@
+# Testing Libraries on WebAssembly
+
+In order to be able to run tests, the following JavaScript engines should be installed:
+- V8
+- JavaScriptCore
+- SpiderMonkey
+
+They can be installed as a part of [jsvu](https://github.com/GoogleChromeLabs/jsvu).
+
+Please make sure that a JavaScript engine binary is available via command line.  
+In case of JSVU, just update PATH variable:
+```bash
+PATH=/Users/<your_user>/.jsvu/:$PATH 
```

Comment:
maybe specify more clearly that you want to prefix this onto the command you run, since running that by itself won't do anything

i.e.

```bash
PATH=/Users/<your_user>/.jsvu/:$PATH your_command
```